src/passes/LoopInvariantCodeMotion.cpp

```diff
@@ -75,7 +75,6 @@ struct LoopInvariantCodeMotion
     // FIXME: also the loop tail issue from above.
     auto numLocals = getFunction()->getNumLocals();
     std::vector<Index> numSetsForIndex(numLocals);
-    std::fill(numSetsForIndex.begin(), numSetsForIndex.end(), 0);
```

Comment:
Creating a `std::vector(size)` with an initial size does already default-initialize to zeros, so here std::fill was redundant.